main.tf

```diff
@@ -157,6 +157,8 @@ module "cloudfront" {
     ssl_support_method       = "sni-only"
     minimum_protocol_version = "TLSv1.2_2021"
   }
+
+  custom_error_response = length(var.custom_error_response) > 0 ? var.custom_error_response : [{}]
```

Comment:
I have added a conditional variable set because the underlying terraform module requires empty object:
https://github.com/terraform-aws-modules/terraform-aws-cloudfront/blob/master/main.tf#L309